README.md

```diff
@@ -40,6 +46,12 @@ load("@io_bazel_rules_scala//scala:scala.bzl", "scala_library", "scala_binary",
 ```
 You may wish to have these rules loaded by default using bazel's prelude. You can add the above to the file `tools/build_rules/prelude_bazel` in your repo (don't forget to have a, possibly empty, BUILD file there) and then it will be automatically prepended to every BUILD file in the workspace.
 
+To run with a persistant worker (much faster), you need to add
+```python
+--strategy=Scalac=worker
+```
+to your command line, or to enable by default for building/testing add it to your .bazelrc.
```

Comment:
to do this do you need to make lines like:
```
build --strategy=Scalac=worker
test --strategy=Scalac=worker
```
or can you just add `--strategy=Scalac=worker`?